install/install_openadapt.ps1

```diff
@@ -55,8 +59,7 @@ function RunAndCheck {
 function Cleanup {
     $exists = Test-Path -Path $setupdir
     if ($exists) {
-        Set-Location $setupdir
-        Set-Location ../
+        Set-Location $env:USERPROFILE
```

Comment:
Should not this be: 

`Set-Location $openAdaptPath` ?

Reply:
Depending on where `Cleanup()` is called from, `$openAdaptPath` might not exist yet, which is why I used `$env:USERPROFILE` here as it is guaranteed to already exist.

Please see my additional clarification [above](https://github.com/OpenAdaptAI/OpenAdapt/pull/449#discussion_r1283464642).

Reply:
Thank you @ayewo 

The idea presented in the table looks great :)